sunpy/map/header_helper.py

```diff
@@ -361,3 +361,52 @@ def _set_instrument_meta(meta_wcs, instrument, telescope, observatory, wavelengt
     'CD2_2':
     'Matrix element CDi_j describing the rotation required to align solar North with the top of the image.'
 }
+
+
+def carrington_header(date, observer_coordinate, *, shape_out, projection_code="CAR"):
+    """
+    Construct a FITS-WCS header for a Carrington coordinate frame.
+    The date-time and observer coordinate of the new coordinate frame
+    are taken from the input map. The resulting WCS covers the full surface
+    of the Sun, and has a reference coordinate at (0, 0) degrees Carrington
+    Longitude/Latitude.
+
+    Parameters
+    ----------
+    date :
+        Date for the output header.
+    observer_coordinate :
+        Observer coordinate for the output header.
+    shape_out : [int, int]
+        Output map shape, number of pixels in (latitude, longitude).
```

Comment:
This description is incorrect

Reply:
Since I swapped the dimensions round to be consistent with the general header helper, I think this is fine now.